tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -559,6 +559,13 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.GraphQLErrorExtensions"/>
         public const string GraphQLErrorExtensions = "DD_TRACE_GRAPHQL_ERROR_EXTENSIONS";
 
+        /// <summary>
+        /// Configuration key for deactivating reading the application monitoring config file through libdatadog (hands off config).
+        /// True by default
+        /// </summary>
+        /// <seealso cref="TracerSettings.GraphQLErrorExtensions"/>
```

Comment:
`GraphQLErrorExtensions` looks like copy/paste error

Reply:
good catch fixed in ce361d118c49a2782db5644bcde134212bf3e5cb